docs/calculations.mdx

```diff
@@ -409,7 +410,7 @@ rendered_width = ad_format.rendered_width_pixels ?? default_device_width[device_
 rendered_height = ad_format.rendered_height_pixels ?? default_device_height[device_type]
 visual_device_coverage =
       is_primary_experience ?
-      device_pixels :
+      1 :
```

Comment:
`rendered_width * rendered_height / device_pixels` is a ration of the screen coverage. This is what we want vs then using the full device w * h. So in the case of a primary experience we use `1` since it is the full screen